src/background/index.ts

```
@@ -934,3 +981,30 @@ function handleGlowStopPort(
 
 // Initialize the extension
 initialize()
+
+// Poll providers when panel is open; compare and broadcast on change
+async function pollProvidersOnce(): Promise<void> {
+  try {
+    const config = await LLMSettingsReader.readAllProviders()
+    const json = JSON.stringify(config)
+    if (json !== lastProvidersConfigJson) {
+      lastProvidersConfigJson = json
+      try { langChainProvider.clearCache() } catch (_) {}
```

ESLint

[error] Empty block statement.

(no-empty)